dev-requirements-py2.txt

```diff
@@ -0,0 +1,15 @@
+beautifulsoup4==4.9.1
+behave==1.2.6
+behaving==1.5.6
+ckanapi==4.3
+factory-boy==2.12.0
```

Comment:
Doesn't this conflict with CKAN 2.8?

The OKFN containers pre-install factory-boy, so I think it would be best not to specify a version here.